### PR TITLE
LowPtElectrons: final ID model weights file

### PR DESCRIPTION
**Pull request by bainbrid**
This PR contains a .root file containing weights for a BDT model used by the LowPtGsfElectronIDProducer module.

- The model is given the label "2020Nov28". 
- Important hyper-parameters: depth=13, trees=1000
- Training details: use 10_6_X UL samples from all 3 years of Run 2
- It replaces the "placeholder" 2020Sept15 model (from BParking studies).

https://github.com/cms-sw/cmssw/pull/32391 depends in this PR.

### CODE REVIEW

**Comment by bainbrid** [comment]
@slava77 @crovelli @cavalari @jainshilpi @lsoffi @guitargeek

**Comment by bainbrid** [comment]
We (i.e. @crovelli!) have investigated various hyperparameter and training configurations for our ID BDT model, with the aim of reducing the disk and memory footprint of the GBRForest. Attached is a plot that shows the performance of various models. Below is a table that summarises the disk footprint of the .root file that would be stored in cms-data.

The "B-Parking" model in the table below is the "2020Sept15" model that is [currently](https://github.com/cms-data/RecoEgamma-ElectronIdentification/commit/e6d91919585cbc139ed1ffeae64afd9d2f08511b#diff-042ac12523d3018f971eaca7dfc9b7b89cef3dd5ea6b4d676fddf914163a2d61) used by the CMSSW_11_3_X, merged from the PR https://github.com/cms-data/RecoEgamma-ElectronIdentification/pull/15. It is planned to be integrated within CMSSW_10_6_X as part of the back port https://github.com/cms-sw/cmssw/pull/32372 for use with the UL reminiAOD compaign. It was trained based on BParking studies and it is our benchmark for all the studies below. It is indicated by the black curve in the attached plot. 

The other models, labeled DepthXX, are indicated in the plot as coloured dashed curves. They are trained on AOD inputs taken from B->J/psi(->ee)K samples produced with 10_6_X.

```
Model       AUC     Footprint (MB)
B-Parking   0.949   39
Depth13     0.960   37
Depth12     0.959   27
Depth10     0.958   8.2
```

The general improvement w.r.t. the baseline "B-Parking" model is understood to be due to the improved reconstruction from the UltraLegacy AOD samples w.r.t. the Prompt conditions used by the BParking samples. 

Our recommendation is to use the Depth10 model, so we plan to update this PR to use the Depth10 model. 

Please let us know if you have any objections.

image

**Comment by bainbrid** [comment]
The last commit implements the changes discussed here https://github.com/cms-data/RecoEgamma-ElectronIdentification/pull/16#issuecomment-740812091

**Comment by slava77** [comment]
+1

based on tests in cms-sw/cmssw#32391
cmsdist and cms-sw/cmssw#32391 will have to be merged for the same IB

**Comment by slava77** [comment]
I'm a bit surprised why the test here passed; 
This PR removes a file supposedly used in the current IB tests.
Did the test pick up the data file from the base release area?

@smuzaffar do we have a mechanism to hide the base release data area ?

**Comment by smuzaffar** [comment]
@slava77 , not currently for data externals but I can update https://github.com/cms-sw/cmssw-config/blob/scramv3/SCRAM/hooks/runtime/50-remove-release-external-lib to drop the data path from release too. let me update build rules and re-run tests for this PR

**Comment by smuzaffar** [comment]
please test with cms-sw/cmsdist#6538

**Comment by silviodonato** [comment]
@smuzaffar , if you agree, I will merge this PR with https://github.com/cms-sw/cmssw/pull/32391 once the tests with cms-sw/cmsdist#6538 are finished

**Comment by smuzaffar** [comment]
please test with cms-sw/cmsdist#6538

**Comment by smuzaffar** [comment]
@silviodonato , yes please wait for the results.

**Comment by silviodonato** [comment]
merge

**Comment by smuzaffar** [comment]
this looks good now, I mean bot now properly remove data path of release. 
@silviodonato , go ahead and merge this with cmssw pr

**Comment by silviodonato** [comment]
@smuzaffar I didn't see the PR from @cmsbuild , so I manually created the new branch (V01-03-00) on cms-data and the PR for cmsdist (cms-sw/cmsdist#6543). I hope everything is ok.

**Comment by smuzaffar** [comment]
you mean new tag V01-04-00 ... right

**Comment by mrodozov** [comment]
the generated release already existed when it tried to create it.
I'm adding a fix for this
